src/applications/pre-need/config/form.jsx

```diff
@@ -81,11 +83,23 @@ function currentlyBuriedPersonsMinItem() {
 }
 
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: `${environment.API_URL}/v0/preneeds/burial_forms`,
   trackingPrefix: 'preneed-',
   transformForSubmit: transform,
   formId: VA_FORM_IDS.FORM_40_10007,
+  saveInProgress: {
+    messages: {
+      inProgress:
+        'Your pre-need determination of eligibility in a VA national cemetery application is in progress.',
+      // TODO: Fix the expired message
+      expired:
+        'Your saved pre-need determination of eligibility in a VA national cemetery application has expired. If you want to apply for pre-need determination of eligibility in a VA national cemetery, please start a new application.',
```

Comment:
What fixing needs to be done?

Reply:
It's just _really_ long and repeats itself. I feel like we should shorten it.